app/forms/decidim/vocdoni/admin/census_permissions_form.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Vocdoni
+    module Admin
+      class CensusPermissionsForm < Decidim::Form
+        mimic :census_permissions
+
+        attribute :census_permissions, Array[String]
```

Comment:
```suggestion
        attribute :verification_types, Array[String]
```
We can rename this so the next assignation is not necessary